TTS/tts/models/vits.py

```diff
@@ -529,14 +641,73 @@ class VitsArgs(Coqpit):
     speaker_embedding_channels: int = 256
     use_d_vector_file: bool = False
     d_vector_dim: int = 0
+
+    # use emotion embeddings
+    use_emotion_embedding: bool = False
+    use_external_emotions_embeddings: bool = False
+    emotions_ids_file: str = None
+    external_emotions_embs_file: str = None
+    emotion_embedding_dim: int = 0
+    num_emotions: int = 0
+    use_text_enc_spk_reversal_classifier: bool = False
+    use_text_enc_emo_classifier: bool = False
+
+    # emotion and speaker embedding squeezer
+    use_emotion_embedding_squeezer: bool = False
```

Comment:
These are in general called ```projection``` not ```squeezer```. There are also squeeze layers but they are different.